src/Picqer/Financials/Exact/SalesInvoice.php

```diff
@@ -1,12 +1,10 @@
-<?php
-
-namespace Picqer\Financials\Exact;
+<?php namespace Picqer\Financials\Exact;
```

Comment:
Thanks! Could you change this back to have the namespace on line 3?

Reply:
Sure thing. I assumed copying the output of the script was what you wanted, maybe change the script too? Or should I put that in a separate pull request?

Reply:
> Or should I put that in a separate pull request?
Please do so. I do not really manage that script, but if the script directly outputs the right code that'd be great :)